deployments/warp_routes/BNB/bsc-zeronetwork-config.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=../schema.json
+tokens:
+  - addressOrDenom: "0xb11d99AD37872068B14b5b9eeb3CE1A4E2244744"
+    chainName: bsc
+    connections:
+      - token: ethereum|zeronetwork|0xa32ea8B73C03eb21A8dC6126D9D0D65D1F8FC65d
+    decimals: 18
+    name: BNB
+    standard: EvmHypNative
+    symbol: BNB
```

Comment:
Recommend adding a logoURI field to all token entries if these will be used in a UI